Apps/W1/External File Storage - Azure Blob Service Connector/app/Entitlements/ExtBlobStorageConnector.Entitlement.al

```diff
@@ -0,0 +1,13 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.ExternalFileStorage;
+
+entitlement "Ext. Blob Storage Connector"
+{
+    Type = Implicit;
+
+    ObjectEntitlements = "Ext. Blob Stor. - Objects";
```

Comment:
in the other entitlements the edit permission set is included.
In this entitlement on the object permission set.

For what do we need thos entitlements here ..?